src/starknet-wrappers.ts

```diff
@@ -265,6 +271,23 @@ export abstract class StarknetWrapper {
 
     public abstract getTxStatus(options: TxHashQueryWrapperOptions): Promise<ProcessResult>;
 
+    protected prepareTransactionTraceOptions(
```

Comment:
Can prepareTxQueryOptions be used here?

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/264/commits/9035f723400261e01d8eee31cc92f913c47f6e4b

Reply:
I think even the argument type can be reused

Reply:
I.e. we might not need the new options type

Reply:
Great catch! https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/264/commits/70a40a06ce8a1305e2fe460d0e78ecbf271e4a2e